app/views/teaching_events/about_git_events.html.erb

```diff
@@ -0,0 +1,101 @@
+  <article>
+    <section class="statement">
+      <span class="pink xlarge bold">What to expect</span>
+    </section>
+
+    <section>
+      <p>Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event.</p>
+
+      <p>They're a series of relaxed and informative events giving you the opportunity to get expert advice and talk to teachers and local training providers.</p>
+
+      <h2 class="heading-m">At the event</h2>
+
+      <p>Drop in any time during an event for presentations, one-to-one chats or to meet others like you who are thinking of stepping into the classroom.</p>
+
+      <p>You can discover:</p>
+
+      <ul>
+        <li>if teaching is right for you</li>
+        <li>how to apply for teacher training</li>
+        <li>what funding is available</li>
+      </ul>
+
+      <h2 class="heading-m">Reserve your place</h2>
+
+      <p>You'll need to register to attend a Get Into Teaching event. Places are limited, so the sooner you sign up, the better.</p>
+    </section>
+
+    <section class="grey-bg fullwidth">
+      <%= render SubjectSpecific::TextBlockWithOneImageComponent.new(
+        heading: "Events in your region",
+        colour: "blue",
+        image_path: "media/images/content/homepage/here-to-help.jpg",
+        image_alt: "Child pointing something out to a teacher in the classroom.",
+      ) %>
+    </section>
+
+    <section class="fullwidth grey-bg no-padding no-margin">
+      <section class="container no-margin">
+        <section class="category__cards">
+          <%= tag.nav(class: "category__nav-cards") do %>
+            <ul class="single-column">
+              <%= render(Categories::CardComponent.with_collection(categorise_events(@git_events))) %>
+            </ul>
+          <% end %>
+        </section>
+      </section>
+    </section>
+
+    <section class="fullwidth grey-bg no-margin">
+      <section class="container no-margin bottom-padding">
+        <div>
+          <h2 class="heading-s">Can't find what you're looking for?</h2>
+          <p><%= link_to("View all events", events_path, class: "link--bold") %></p>
+        </div>
+      </section>
+    </section>
+
+    <section class="what-theyre-saying fullwidth no-margin no-padding">
+      <div class="triangle grey-bg yellow-triangle">
+        <svg height="100%" width="100%" viewBox="0 0 100 100" preserveAspectRatio="none">
+          <polygon points="0,100 100,0 100,100"></polygon>
+        </svg>
+      </div>
+      <div class="subheading">
+        <h3 class="heading--box-pink heading--margin-0">What they're saying</h3>
+      </div>
+      <figure>
+        <blockquote>I had doubts on what to do and how to proceed, but have found answers to all of them thanks to this event.</blockquote>
+      </figure>
+    </section>
+
+    <section class="fullwidth">
+      <section class="container">
+        <span class="blue xlarge bold">See what you get out of an event</span>
```

Comment:
@GDW7 were you looking to shorten / change this heading? the change from the other PR isn't in this one

```suggestion
        <span class="blue xlarge bold">Why you should attend</span>
```